Admin-app/src/backend/firebaseAdminUtilities.js

```diff
@@ -49,10 +49,46 @@ const getAdminRef = (key) => {
   return getDocRef(getTableName(), key);
 };
 
+const setDisabled = async (patientKey) => {
+  console.log("[setStatus]" + patientKey);
```

Comment:
This should be taken out (copy/paste probably).

Reply:
Yes, I didn't notice thank you